config/manager/manager.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: dspipelines-controller
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: ds-pipelines-controller
+    app.kubernetes.io/part-of: ds-pipelines-controller
+spec:
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  replicas: 1
```

Comment:
Can we bump this up to 3?

Reply:
We're already passing leader-elect

Reply:
Should we also introduce pod anti-affinities?

Reply:
> Can we bump this up to 3?
 done
>  Should we also introduce pod anti-affinities?
yes we should, this is a feature request though should be a github issue